2/scss/footer.scss

```diff
@@ -0,0 +1,247 @@
+    text-align: center;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    &-icons {
+      display: grid;
+      grid-template-columns: 100% 100%;
+      padding-left: 0;
+      list-style-type: none;
+      .twitter {
+        margin-right: 70px;
+        margin-bottom: 40px;
+        cursor: pointer;
+        display: inline-block;
+        width: 30px;
+        height: 24px;
+        background-image: url(../images/icons.png);
+        background-position: 0 0;
+        &:hover {
+          background-position: -30px 0px;
+          transition: 300ms background-position;
+        }
+      }
+      .facebook {
+        cursor: pointer;
+        display: inline-block;
+        width: 25px;
+        height: 25px;
+        background-image: url(../images/icons.png);
+        background-position: 54px 0;
+        &:hover {
+          background-position: 29px 0;
+          transition: 300ms background-position;
+        }
+      }
+      .youtube {
+        margin-right: 70px;
+        cursor: pointer;
+        display: inline-block;
+        width: 32px;
+        height: 23px;
+        background-image: url(../images/icons.png);
+        background-position: 0 24px;
+        &:hover {
+          background-position: -32px 24px;
+          transition: 300ms background-position;
+        }
+      }
+      .in {
+        cursor: pointer;
+        display: inline-block;
+        width: 25px;
+        height: 25px;
+        background-image: url(../images/icons.png);
+        background-position: 50px 25px;
+        &:hover {
+          background-position: 25px 25px;
+          transition: 300ms background-position;
+        }
+      }
+    }
+  }
+  .skout-bottom {
+    max-width: 105px;
+    max-height: 38px;
+  }
+}
+
+@media (min-width: 768px) {
+  .footer {
+    &-wrapper {
+      padding: 28px 54px 50px;
+      flex-direction: column;
+      .social {
+        justify-content: space-between;
+        grid-template-columns: none;
+        padding-top: 30px;
+        &-icons {
+          display: flex;
+          grid-template-columns: 0;
+          .social-icon {
+            margin-right: 25px;
+          }
+          .twitter {
+            margin-right: 0;
+            margin-bottom: 0;
+          }
+          .youtube {
+            margin-right: 0;
+          }
+        }
+      }
+      .address-container {
+        .address {
+          .location-container {
+            display: flex;
+            flex-direction: column;
+          }
+        }
+      }
+    }
+  }
+}
+
+@media (min-width: 1440px) {
+  .footer {
+    &-wrapper {
+      margin: auto;
+      padding: 75px 135px 110px;
+      .about-company {
+        display: flex;
+        justify-content: space-between;
+        padding-top: 78px;
+        flex-direction: row;
+        .link-company-container {
+          padding-right: 100px;
+          .link-company-item {
+            margin-bottom: 30px;
+            .link-company {
```

Comment:
bad class name. In this case you code remove class and use just `a`

better when you start from short classes and add additional inside:
```
.link-company {
  ...

  &-container {
    ...
  }

  &-item {
    ...
  }
}
```

Reply:
renamed